src/librados/IoCtxImpl.h

```diff
@@ -158,7 +158,8 @@ struct librados::IoCtxImpl {
   int operate_read(const object_t& oid, ::ObjectOperation *o, bufferlist *pbl, int flags=0);
   int aio_operate(const object_t& oid, ::ObjectOperation *o,
 		  AioCompletionImpl *c, const SnapContext& snap_context,
-		  int flags, const blkin_trace_info *trace_info = nullptr);
+		  ceph::real_time *pmtime, int flags,
```

Comment:
nit: how about declaring this as a pointer-to-constant?

```suggestion
		  const ceph::real_time *pmtime, int flags,
```